src/config.xml

```diff
@@ -2600,6 +2600,15 @@ EXTRA_SEARCH_MAPPINGS = tagname1=loc1 tagname2=loc2 ...
  docs will be put.
  If a relative path is entered the value of \ref cfg_output_directory "OUTPUT_DIRECTORY" will be
  put in front of it.
+]]>
+      </docs>
+    </option>
+    <option type='string' id='PDF_DST_FILE' format='file' defval='' depends='GENERATE_LATEX'>
+      <docs>
+<![CDATA[
+ The \c PDF_DST_FILE copies the generated refman.pdf to the defined destination. This makes the
+ PDF much easier to access than in the full blown latex build folder. It will take effekt at the
+ end of the makefile run. It is a path relative to the build folder of latex.
```

Comment:
Seen the implementation I think the directory can either be a full path and otherwise it is a path relative to the build folder of latex.